src/lib.rs

```diff
@@ -152,6 +153,22 @@ pub fn that(path: impl AsRef<OsStr>) -> io::Result<()> {
     Err(last_err.expect("no launcher worked, at least one error"))
 }
 
+pub fn these(
+    path: impl IntoIterator<Item = impl AsRef<OsStr>> + std::convert::AsRef<std::ffi::OsStr>,
```

Comment:
There is no need for any function that can be emulated with `for path in paths { open::that(path)?; }` Instead, in order to implement it as described in #70, launchers must be fed multiple paths if they support it.